queries/go/highlights.scm

```diff
@@ -15,9 +16,6 @@
 ((identifier) @constant
  (#eq? @constant "_"))
 
-((identifier) @constant
- (#vim-match? @constant "^[A-Z][A-Z\\d_]+$"))
```

Comment:
Can you describe your motivation to remove this? Where does it produce bad highlighting for you? This help others to understand your changes better

Reply:
I posted a screenshot of what it does and it is completely wrong for golang. There is no such rule to uppercase the constants in golang, it is just a normal identifier and you cannot highlight them without using some kind of scope.